README.md

```diff
@@ -1,340 +1,535 @@
 
----
+### Deposit and register Validator
 
-### Security notes
+When the `ssv-dkg` tool is launched as shown above, it will commence a DKG ceremony with the selected operators, which will end in the creation of two files:
+* a deposit JSON file - necessary to perform the transaction on the Deposit contract and activate the validator on the Beacon layer
+* a key shares JSON file - necessary to register the validator on the ssv.network
+### Troubleshooting
 
-Here we explain how we secure the communication between DKG ceremony initiator and operators
+#### dial tcp timeout
+```sh
+2023-10-11T16:36:26.745937Z     FATAL   dkg-initiator   😥 Failed to initiate DKG ceremony:     {"error": "Post \"http://79.44.117.213:3030/init\": dial tcp 79.44.117.213:3030: i/o timeout"}
+```
+When this error appears, it means that the `ssv-dkg` tool cannot connect to one of the selected operators.
+This could be temporary, but if it persists, we recommend changing one of the operators.
+#### invalid URI for request
+```sh
+2023-10-11T16:29:47.226138Z     FATAL   dkg-initiator   😥 Failed to load operators:    {"error": "invalid operator URL parse \"80.181.85.114:3030\": invalid URI for request"}
+```
+When this error appears, it means that the endpoint information for one of the operators is incorrect.
+You could manually verify the `operators_info.json` or the initiator command-generated by the webapp, or simply change one of the operators.
+#### connection refused
+```sh
+2023-10-13T15:21:54.597429Z     FATAL   dkg-initiator   😥 Failed to initiate DKG ceremony:     {"error": "Post \"http://80.181.85.114:3030/init\": dial tcp 80.181.85.114:3030: connect: connection refused"}
+```
+When this error appears, it means that the `ssv-dkg` tool cannot connect to one of the selected operators, and the reason could be because their `ssv-dkg` operator node has shut down.
+This could be temporary, as they will likely start the node again, but if it persists, we recommend changing one of the operators.
 
-1. Initiator is using RSA key (2048 bits) to sign init message sent to operators. Upon receiving operators verify the sig using pub key at init message. If the sig is valid, operators store this pub key for further verification of messages coming from the initiator(s).
-2. Operators are using RSA key (ssv operator key - 2048 bits) to sign every message sent back to initiator.
-3. Initiator verifies every message incoming from any operator using ID and Public Key provided by operators info file, then initiator creates a combined message and signs it.
-4. Operators verify each of the messages of other operators participating in the ceremony and verifies initiator`s signature of the combined message.
-5. During the DKG protocol execution, the BLS auth scheme is used - G2 for its signature space and G1 for its public keys
 
-## Architecture
+## Operator Quick start
 
-![flow](./docs/imgs/DKGinit.drawio.png)
```

Comment:
maybe we should move flow and architecture to a different file. its worth to have a note about the arch and design for more advanced users. but its not a must in the readme.